### PR TITLE
Mail database

### PR DESCRIPTION
**Pull request by lisperata**
image
image
image
image

### CODE REVIEW

File: sequelize/contactModel.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import pkg from "sequelize";
+const { DataTypes } = pkg;
+import sequelize from "./index";
+
+const contactSequelize = sequelize.define("contacts", {
+  uuid: {
+    type: "VARCHAR(36)",
```
Comment (CatDidenko) [inline sequelize/contactModel.ts]: Should be uuidv4 type and autogenerated

File: sequelize/listModel.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import pkg from "sequelize";
+const { DataTypes } = pkg;
+import sequelize from "./index";
+
+const listSequelize = sequelize.define("contact", {
+  uuid: {
+    type: "VARCHAR(36)",
```
Comment (CatDidenko) [inline sequelize/listModel.ts]: Should be uuidv4 type and autogenerated

File: sequelize/listModel.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import pkg from "sequelize";
```
Comment (CatDidenko) [inline sequelize/listModel.ts]: Bad naming

**Review by CatDidenko — CHANGES_REQUESTED** [review]
General incorrect things:
1) We should use module project structure. It mean all files related to one entity, should be in the same folder. 
Example: 
folder contact  should includes contacts controller, contacts service, contacts route and contacts model.
2) models/Contact.ts - It's not a model. It's a service, not a model file. Model describes an entity, not actions on it.
3) model/List.ts - it's not a model...
4) Don't get attached to the name of ORM that you use. Rename to more general - database. Also database folder should includes only configs for database. Models must be located in the appropriate modules. (contact model in contact folder)

File: sequelize/contactModel.ts (hunk 1 above)
Comment (lisperata) [reply sequelize/contactModel.ts]: Used autogeneration

File: sequelize/listModel.ts (hunk 2 above)
Comment (lisperata) [reply sequelize/listModel.ts]: Used autogeneration

File: sequelize/listModel.ts (hunk 3 above)
Comment (lisperata) [reply sequelize/listModel.ts]: Renamed

File: contact/сontactService.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    );
+  }
+
+  public async changeContactById(): Promise<void> {
+    const { uuid, name, email } = this;
+    const nameForQuery = name
+      ? this._convertParameterToQuery("name", name)
+      : "";
+    const emailForQuery = email
+      ? this._convertParameterToQuery("email", email)
```
Comment (CatDidenko) [inline contact/сontactService.ts:56]: Please, use single quotes

**Review by CatDidenko — APPROVED** [review]

**Pull request merged**